tests/integration-tests/tests/storage/test_ebs.py

```diff
@@ -20,37 +20,64 @@
 from tests.storage.kms_key_factory import KMSKeyFactory
 from tests.storage.storage_common import (
     assert_subnet_az_relations_from_config,
+    test_directory_correctly_shared_between_ln_and_hn,
     test_ebs_correctly_mounted,
     verify_directory_correctly_shared,
 )
 
 
+@pytest.mark.parametrize("use_login_node", [True, False])
```

Comment:
General comment that holds for all tests: 

I initially hoped that we could reuse the existing test and simply parameterize it.
However seeing the changes you had to make this is not possible and you were forced to add a new section with custom operations to run on the login nodes.

This makes the parameterization counterproductive because we are running the test twice (use_login_node=[False, True] ) and
 - in the first run we are only executing the tests on the HeadNode
 - in the second run we are executing the test both on the HeadNode AND the LoginNodes
 
This means that we are running the test in the HeadNode twice (without changes) and that we are creating two clusters without a real advantage.

So I propose to remove the parameterization 

```
@pytest.mark.parametrize("use_login_node", [True, False])
```
from all the tests and simply leave the extra section `LoginNodes` in the configs and the extra checks for the LoginNodes at the end of the tests.

This will allow us to test both HeadNode and LoginNode with a single cluster.

Reply:
Makes sense. Done!